docker/wrapper.py

```diff
@@ -285,12 +417,25 @@ def main():
     for samples in [args.sample_tar, args.sample_paired, args.sample_single]:
         if not samples:
             continue
-        # If sample is given as relative path, assume it's in the work directory
-        if not all(x.startswith('/') for x in samples):
-            samples = [os.path.join(work_mount, x) for x in samples if not x.startswith('/')]
-            log.info('\nSample given as relative path, assuming sample is in work directory: {}'.format(work_mount[0]))
+
+        if not args.auto_scale:
```

Comment:
Put in an `else` block with `if args.auto_scale`

Reply:
done